conf/app.routes

```diff
@@ -12,6 +12,9 @@ GET         /eori/eori-history                          controllers.EoriHistoryC
 GET         /eori/:eori/xieori-information              controllers.XiEoriController.getXiEoriInformation(eori: String)
 GET         /eori/xieori-information                    controllers.XiEoriController.getXiEoriInformationV2()
 
+GET         /eori/verified-email-third-party           controllers.VerifiedEmailController.retrieveVerifiedEmailThirdParty()
+GET         /eori/company-information-third-party      controllers.CompanyInformationController.retrieveCompanyInformationThirdParty()
```

Comment:
It should be POST. Ticket should have been updated. Will be updated very shortly.